x-pack/plugins/rule_registry/common/types.ts

```diff
@@ -287,11 +279,22 @@ export const metricsAggsSchemas = t.exact(
         }),
       })
     ),
-    aggs: t.undefined,
```

Comment:
what Xavier did here? We did that to not allow user to create nested aggs on metrics aggs knowing that ES won't like it. It was to make the API nicer. However, this created a nightmare to combine metrics and bucket aggs together. So We decide to remove these two attribute sand let ES tell you when the aggs is not formatted correctly.